media/css/protocol/components/_footer.scss

```diff
@@ -39,75 +39,6 @@ $image-path: '/media/protocol/img';
     }
 }
 
-// * -------------------------------------------------------------------------- */
-// Light theme (for Firefox pages)
-
-.mzp-c-footer.mzp-t-light {
-    background: $color-marketing-gray-10;
-    color: $color-black;
-
-    a:link,
-    a:visited {
-        color: $color-marketing-gray-80;
-
-        &:hover,
-        &:focus,
-        &:active {
-            color: $color-marketing-gray-90;
-        }
-    }
-
-    .mzp-c-footer-sections {
-        border-bottom-color: $color-marketing-gray-30;
-    }
-
-    .mzp-c-footer-primary-logo a {
-        background-image: url('#{$image-path}/logos/mozilla/logo-word-hor.svg');
-    }
-
-    .mzp-c-footer-links-social li a {
-        &.twitter {
-            background-image: url('#{$image-path}/icons/social/x/black.svg');
-        }
-
-        &.instagram {
-            background-image: url('#{$image-path}/icons/social/instagram/black.svg');
-        }
-
-        &.youtube {
-            background-image: url('#{$image-path}/icons/social/youtube/black.svg');
-        }
-
-        // bedrock override until icons are added to protocol-assets
-        // https://github.com/mozilla/protocol-assets/issues/84
-        &.linkedin {
-            background-image: url('/media/img/logos/social/linkedin-black.svg');
-        }
-
-        &.tiktok {
-            background-image: url('/media/img/logos/social/tiktok-black.svg');
-        }
-
-        &.spotify {
-            background-image: url('/media/img/logos/social/spotify-black.svg');
-        }
-
-        &.mastodon {
-            background-image: url('/media/img/logos/social/mastodon-black.svg');
-        }
```

Comment:
The "black" versions are now not linked from anywhere. Not that they _must_ be removed as they are handy generally… but with:
- https://github.com/mozilla/protocol-assets/pull/85
- https://github.com/mozilla/protocol-assets/pull/88

i think now not only the "black", but also "white" could be eventually cleaned up from all the codebase, and used directly from Protocol. Does anything come to mind blocking this? _(I might update the local overrides for footer to `$image-path` after this PR lands, and remove the assets there…)_

Reply:
Ah good call, thank you! I removed icons now :)